dashboard/test/controllers/activities_controller_test.rb

```diff
@@ -886,9 +759,7 @@ def new_level
     assert_equal true, new_level
     assert_equal false, new_level
 
-    sign_out @user
```

Comment:
Even for anonymous users we should try to POST to milestone and get back the same new level response (I think), please let me know if that's not the case.